dbclient/WorkspaceClient.py

```diff
@@ -176,9 +178,18 @@ def export_user_home(self, username, local_export_dir):
         if num_of_nbs != num_of_nbs_dl:
             print(f"Notebooks logged != downloaded. Check the failed download file at: {user_export_dir}")
         print(f"Exporting the notebook permissions for {username}")
-        self.log_acl_to_file('notebooks', 'user_workspace.log', 'acl_notebooks.log', 'failed_acl_notebooks.log')
+        acl_notebooks_writer = ThreadSafeWriter("acl_notebooks.log", "w")
+        try:
+            self.log_acl_to_file('notebooks', 'user_workspace.log', acl_notebooks_writer, 'failed_acl_notebooks.log')
```

Comment:
we just have to do the error_logger since python logging libraries are already thread-safe: https://docs.python.org/2/library/logging.html#thread-safety